### PR TITLE
Fix : Service Mesh Test (QE)

### PR DESCRIPTION
**Pull request by bnshr**

### CODE REVIEW

File: tests/platformalteration/tests/platform_alteration_service_mesh.go [hunk 1]
```diff
@@ -35,15 +42,12 @@ var _ = Describe("platform-alteration-service-mesh-usage", func() {
 
 	// 56594
 	It("istio is installed", func() {
-		By("Create istio-system namespace")
-		err := namespaces.Create(istioNs, globalhelper.APIClient)
-		Expect(err).ToNot(HaveOccurred())
-
+		By("Define a test pod with instio container")
```
Comment (theemj) [inline tests/platformalteration/tests/platform_alteration_service_mesh.go:45]: I suspect this should be "istio"

File: tests/platformalteration/tests/platform_alteration_service_mesh.go [hunk 2]
```diff
@@ -53,30 +57,15 @@ var _ = Describe("platform-alteration-service-mesh-usage", func() {
...
-		Expect(err).ToNot(HaveOccurred())
 	})
 
 	// 56596
 	It("istio is installed but proxy containers does not exist [negative]", func() {
-		By("Create istio-system namespace")
-		err := namespaces.Create(istioNs, globalhelper.APIClient)
-		Expect(err).ToNot(HaveOccurred())
-
+		By("Define a test pod without instio container")
```
Comment (theemj) [inline tests/platformalteration/tests/platform_alteration_service_mesh.go:64]: Another "instio"

File: tests/platformalteration/tests/platform_alteration_service_mesh.go [hunk 3]
```diff
@@ -86,27 +75,22 @@ var _ = Describe("platform-alteration-service-mesh-usage", func() {
...
-
 	})
 
 	// 56597
 	It("istio is installed but proxy container exist on one pod only [negative]", func() {
-		By("Create istio-system namespace")
-		err := namespaces.Create(istioNs, globalhelper.APIClient)
-		Expect(err).ToNot(HaveOccurred())
-
 		By("Define first pod with instio container")
```
Comment (theemj) [inline tests/platformalteration/tests/platform_alteration_service_mesh.go:82]: Pre-existing origin of the "instio" typo lol

**Review by theemj — APPROVED** [review]

**Comment by theemj** [comment]
Couple typos in informational lines but otherwise looks great and I tested it on my cluster and it works great as well!

File: tests/platformalteration/tests/platform_alteration_service_mesh.go (hunk 1 above)
Comment (bnshr) [reply tests/platformalteration/tests/platform_alteration_service_mesh.go]: Done!

**Review by greyerof — APPROVED** [review]

**Pull request merged**